logistics/scripts/modules/database/postgres-database-management.psm1

```diff
@@ -296,6 +296,37 @@ function Remove-PostgreSQLDatabase {
     Test-Error
 }
 
+function Get-SandboxDatabaseNames {
+    Param (
+        [string] [Parameter(Mandatory = $true)] $serverName,
+
+        [string] [Parameter(Mandatory = $true)] $portNumber,
+
+        [string] $userName
+    )
+
+    $parameters = @{
+        serverName   = $serverName
+        portNumber   = $portNumber
+        userName     = $userName
+        databaseName = 'postgres'
+        commands     = "select datname from pg_database where datname similar to 'EdFi_Ods_Sandbox%';"
```

Comment:
we need to be using the $templateCSB so that the `EdFi_` prefix is configurable. Think about what this would do in our staging environment for example.

Reply:
Done